### PR TITLE
Update installation instructions for AudienceTargetingBundle

### PR DESCRIPTION
**Pull request by nnatter**
| Q | A
| --- | ---
| License | MIT

#### What's in this PR?

This PR updates the installation instructions for the `AudienceTargetingBundle`.

### CODE REVIEW

File: bundles/audience_targeting.rst [hunk 1]
```diff
@@ -12,42 +12,25 @@ target group of the visitor.
...
-            ];
-
-            return $bundles;
-        }
-    }
+    Sulu\Bundle\AudienceTargetingBundle\SuluAudienceTargetingBundle::class => ['all' => true],
 
 Add the routes for the administration interface to the routing configuration
-file (`app/config/admin/routing.yml`):
+file (`config/routes/sulu_admin.yaml`):
```
Comment (alexander-schranz) [inline bundles/audience_targeting.rst]: if we want to provide a recipe for this bundle we should move this routes into an own config file e.g: `sulu_audience_targeting_admin.yaml`

File: bundles/audience_targeting.rst [hunk 2]
```diff
@@ -12,42 +12,25 @@ target group of the visitor.
...
-        resource: "@SuluAudienceTargetingBundle/Resources/config/routing.xml"
-        prefix: /admin/target-groups
-
     sulu_audience_targeting_api:
         resource: "@SuluAudienceTargetingBundle/Resources/config/routing_api.xml"
         prefix: /admin/api
 
 And the routes for the website in the corresponding configuration file
-(`app/config/website/routing.yml`):
+(`config/routes/sulu_website.yaml`):
```
Comment (alexander-schranz) [inline bundles/audience_targeting.rst]: same as above would add `sulu_audience_targeting_website.yaml`

File: bundles/audience_targeting.rst (hunk 1 above)
Comment (danrot) [reply bundles/audience_targeting.rst]: I don't know if I would do that now.

1. If we do that, then the entire `sulu_admin.yaml` file should be gone.
2. If we support Symfony Flex, then these steps would be gone completely anyway.

Comment (alexander-schranz) [reply bundles/audience_targeting.rst]: correct we have all other routes in the sulu_admin.yaml as well

File: bundles/audience_targeting.rst [hunk 3]
```diff
@@ -12,62 +12,48 @@ target group of the visitor.
...
 .. code-block:: yaml
 
     # ...
-    sulu_audience_targeting:
-        resource: "@SuluAudienceTargetingBundle/Resources/config/routing.xml"
-        prefix: /admin/target-groups
-
     sulu_audience_targeting_api:
-        resource: "@SuluAudienceTargetingBundle/Resources/config/routing_api.xml"
+        resource: "@SuluAudienceTargetingBundle/Resources/config/routing_api.yml"
```
Comment (alexander-schranz) [inline bundles/audience_targeting.rst]: @nnatter is here `type: rest` missing? for be compatible with symfony 3.4 name_prefix?

**Pull request merged**